docs/install.md

```diff
@@ -41,14 +41,16 @@ When everything is ready, navigate to your forum in a web browser and follow the
 
 ### Apache
 
-Flarum includes a `.htaccess` file in the `public` directory – make sure it has been uploaded correctly. If you're using shared hosting, confirm with your provider that `mod_rewrite` is enabled and `.htaccess` files are allowed. If you're managing your own server, you may need to add the following to your site configuration:
+Flarum includes a `.htaccess` file in the `public` directory – make sure it has been uploaded correctly. **Flarum will not function properly if `mod_rewrite` is not enabled and `.htaccess` is not allowed.** Be sure to check with your hosting provider (or your VPS) that these features are enabled. If you're managing your own server, you may need to add the following to your site configuration to enable `.htaccess` files:
```

Comment:
> `mod_rewrite` is not enabled and `.htaccess` is not allowed

IMO, that should be:

> `mod_rewrite` is not enabled or `.htaccess` is not allowed

That said, what about adding a line or two about enabling `.htaccess` overrides? If I remember correctly, this isn't enabled by default anymore in recent versions of Apache.

Reply:
Done. How's it look now?

Reply:
Nice, thank you!